common/SmokeTests/SmokeTest/KeyVaultTest.cs

```diff
@@ -32,7 +34,20 @@ public static async Task RunTests()
             string clientID = Environment.GetEnvironmentVariable("APP_CLIENT_ID");
             string clientSecret = Environment.GetEnvironmentVariable("CLIENT_SECRET");
             string keyVaultUri = Environment.GetEnvironmentVariable("KEY_VAULT_URI");
-            client = new SecretClient(new Uri(keyVaultUri), new ClientSecretCredential(tenantID, clientID, clientSecret));
+            
+            var authorityHost = Environment.GetEnvironmentVariable("AZURE_AUTHORITY_HOST") 
```

Comment:
Should we just require this even for the default cases now?

Reply:
What do you mean by that wes?

Reply:
In the interest of getting this merged I don't think this is a blocker. We currently have a "safe" default to the public cloud for 3 languages for the smoke test including this one. I've opened this issue so we can close up that discussion: https://github.com/Azure/azure-sdk/issues/1104